app/src/main/java/com/yagni/view/HomeGUI.java

```diff
@@ -1,136 +1,119 @@
 package com.yagni.view;
-
-import java.awt.BorderLayout;
-import java.awt.Color;
-import java.awt.Dimension;
-import java.awt.Desktop; 
-import java.awt.FlowLayout;
-import java.awt.Font;
-import java.awt.FontFormatException;
-import java.awt.GraphicsEnvironment;
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
+import java.awt.*;
```

Comment:
There should be a vertical spacer between the package statement and the import statements(b/w line 1/2)